intervals/interval.py

```diff
@@ -86,11 +86,11 @@ def canonicalize(interval, lower_inc=True, upper_inc=False):
 def coerce_interval(func):
     def wrapper(self, arg):
         if (
-            isinstance(arg, list) or
-            isinstance(arg, tuple) or
-            isinstance(arg, self.type) or
-            isinstance(arg, type(self)) or
-            arg == inf or arg == -inf
+                isinstance(arg, list) or
```

Comment:
Please remove this and all similar style changes where you add extra indents.

Reply:
I have revise all that changes.